docs/data/core-blocks.md

```diff
@@ -0,0 +1,86 @@
+# **core/blocks**: Block Types Data
+
+## Selectors 
+
+### getBlockType
+
+Returns a block type by name.
+
+*Parameters*
+
+ * state: Data state.
+ * name: Block type name.
+
+
+### getCategories
+
+Returns all the available categories.
+
+*Parameters*
+
+ * state: Data state.
+
+*Returns*
+
+Categories list.
+
+### getDefaultBlockName
+
+Returns the name of the default block name.
```

Comment:
The default block name in what context? I'm guessing it's for the editor context (and this is probably `<p>` by default), but that isn't clear from just reading the docs.